environment.yml

```diff
@@ -23,3 +23,8 @@ dependencies:
   # Multi language support:
   - ncl>=6.5.0  # this should always install 6.6.0 though
   - r-base>=3.5
+  # R packages needed for development
+  - r-docopt
+  - r-lintr
+  - r-styler
+  - r-yaml
```

Comment:
that's great this removes the compilation stage of these packages - will the other packages (like akima, etc) need to be compiled still? How much of a hit we're taking from extra packages in the conda env and the possibility the env will now be harder to solve? (I am not expecting an answer to the last question since that really depends on conda weather, but just a precautionary thought) :beer:

Reply:
Yes, the other packages (at least the ones that are not pulled in as dependencies of these packages) will still need to be compiled. Indeed the environment will become slightly harder to solve. It would be possible to add many more R packages, because most of them are available on conda, but that would definitely make the environment harder to solve, especially when building the package from package/meta.yml.

Reply:
well, it's a question of the lesser evil I guess